crates/holochain/src/conductor/api/api_external/admin_interface.rs

```diff
@@ -291,6 +291,13 @@ impl AdminInterfaceApi for RealAdminInterfaceApi {
             StorageInfo => Ok(AdminResponse::StorageInfo(
                 self.conductor_handle.storage_info().await?,
             )),
+            InitializeDeepkey { deepkey_dna } => {
+                self.conductor_handle
+                    .clone()
+                    .initialize_deepkey(Some(deepkey_dna))
+                    .await?;
+                Ok(AdminResponse::Ok)
+            }
```

Comment:
This doesn't seem right. Shouldn't this be `InitializeDpki { dpki_dna }` so that we can initialize dpki services that are not deepkey?

Reply:
yeah, I changed this is in a subsequent PR. I agree.